### PR TITLE
Racial Redux Part One

### PR DESCRIPTION
**Pull request by Kroto-Ker**
Let's get racial.

## About The Pull Request

Adjusts the lore of Humens, Elves, Half-Elves, Wild-Kin, and Half-Kin to match our setting. Racial descriptions for these races have been changed.

Humen & Half-Kin ancestries have been changed to: Frostlander, Umberite, Grenzelhoft, Rockhill, Heartfell, Highlander, Moravian, Forester, Zybantine, Merkite, and Valorian.

Elf tribal identities have been changed to: Snow Scion, Shadewood, Emberfall, Vandendor, Fablefield, Nevor, Merkite, and Weso.

Half-Elf identities have been changed to: Frost-Scion, Shade-Forester, Fable-Hoft, Zybantine-Nevor, and Valorian-Born.

Wild-Kin now have selectable habitat identities! These are: the Grand Forest, the Crimson lands, the Terrorbog, the Highlands, the Western Mountains, the Frostlands, the Unknown Lands, the Valorian Deserts, and the Eastern Plains.
These all correlate to areas on our world map and allow for further customization!

## Why It's Good For The Game

Brings our lore into the game itself! Epic!

### CODE REVIEW

**Comment by Stutternov** [comment]
Just to note, strength and speed are easily worth more than end, con, or intelligence. Giving male halfkin bonus strength should warrant another negative.

**Comment by Kroto-Ker** [comment]
> Just to note, strength and speed are easily worth more than end, con, or intelligence. Giving male halfkin bonus strength should warrant another negative.

Male half-kin don't get bonus strength.

**Comment by Kroto-Ker** [comment]
Removed stat changes since Rebel0 should be porting stat paks tonight anyways.

File: code/modules/mob/living/carbon/human/species_types/furry/anthromorph.dm [hunk 1]
```diff
@@ -4,10 +4,18 @@
 /datum/species/anthromorph
 	name = "Wild-Kin"
 	id = "anthromorph"
-	desc = "Wild-Kin; a name for a furred creacher that does not fall under any particular race. \
-	It is sometimes difficult to tell one of us from another race at a glance; one of Wild-Kin ancestry may look very similar to a Tabaxi or Vulpkanin. \
-	Due to the fact we are so numerous, we are welcome almost anywhere, along with our Half-Kin sister-race \
-	We are average of constitution, strength, and intellegence."
+	desc = "<b>Wild-Kin</b><br>\
+	The Wild Volk are the errant children of Dendor, to whom they all share an innate connection. \
```
Comment (LonelyHowls) [inline code/modules/mob/living/carbon/human/species_types/furry/anthromorph.dm:8]: Change this to "Wild Folk"

File: code/modules/mob/living/carbon/human/species_types/furry/anthromorph.dm [hunk 2]
```diff
@@ -4,10 +4,18 @@
...
+	Wild-Kin arise in all the lands of Grimoria and are nearly as widespread as Humens.<br>\
+	Wild-Kin are humanoid animals, sometimes limited to one specific species or a hybrid of multiple. \
+	Of all the races, Wild-Kin has the most variety in appearance.<br>\
```
Comment (LonelyHowls) [inline code/modules/mob/living/carbon/human/species_types/furry/anthromorph.dm:11]: This *works* but if we want it to be a little better I'd change it to "Of all of the races, Wild-Kin have the most variety in their appearance."

File: code/modules/mob/living/carbon/human/species_types/furry/demihuman.dm [hunk 3]
```diff
@@ -4,10 +4,17 @@
 /datum/species/demihuman
 	name = "Half-Kin"
 	id = "demihuman"
-	desc = "Half-Kin, earth-borne, magyck-touched. Many names exist for my race, characterized by our varied appearance. \
-	Wild-Kin are by far the most numerous people due to the fact we are, usually, compatible with almost all other races. \
-	Due to the fact we are so numerous, we are welcome almost anywhere, along with our Wild-Kin sister race \
-	We are average of constitution, strength, and intellegence."
+	desc = "<b>Half-Kin</b><br>\
+	A halfbreed that arises in Humens with Wild-Kin in their lineage. \
```
Comment (LonelyHowls) [inline code/modules/mob/living/carbon/human/species_types/furry/demihuman.dm:8]: "half-breed" is the technically correct way of writing this.

File: code/modules/mob/living/carbon/human/species_types/furry/demihuman.dm [hunk 4]
```diff
@@ -4,10 +4,17 @@
...
+	They are children of Dendor by blood, but their Humen heritage makes this connection powerless. \
```
Comment (LonelyHowls) [inline code/modules/mob/living/carbon/human/species_types/furry/demihuman.dm:9]: I'd change the wording here.

"makes this connection tenuous at best" or something to that effect works better. "Makes this connection powerless" feels off?

File: code/modules/mob/living/carbon/human/species_types/furry/demihuman.dm [hunk 5]
```diff
@@ -4,10 +4,17 @@
...
+	Half-Kin are said to be the only ones of Dendor’s kin that can resist his Call of the Wild.<br>\
```
Comment (LonelyHowls) [inline code/modules/mob/living/carbon/human/species_types/furry/demihuman.dm:10]: I'd change it to read as

Half-Kin are said to be the only of Dendor's progeny who can resist* 

It reads a little better

File: code/modules/mob/living/carbon/human/species_types/roguetown/elf/elfs.dm [hunk 6]
```diff
@@ -5,17 +5,14 @@
...
-	Elves tend to be looked poorly upon by humans, as historically the two races have \
-	been rivals in various conflicts and territorial disputes. This however does not stop \
-	many humans and elves from forming relationships, which are capable of producing child.\
-	Elves are known for their intelligence and sharp eyes, but their graceful nature does \
-	not lend itself to the concepts of strength or durability... \
-	There are elves from a small smattering of tribes in these parts."
+	The Elves are marked apart from other races by their ethereal grace and long lives. \
+	Like Humens, they live throughout the lands of Grimoria. \
+	Yet, unlike other races, they prefer to dwell in lands untouched by civilization.<br>\
+	Elves can be hard to tell apart from Humens at a glance but can be easily distinguished once acquainted. \
```
Comment (LonelyHowls) [inline code/modules/mob/living/carbon/human/species_types/roguetown/elf/elfs.dm:11]: This just feels sloppy.

Suggestion:

"Elves can be difficult to distinguish from Humens at a glance until one is better acquainted with them."

File: code/modules/mob/living/carbon/human/species_types/roguetown/elf/elfs.dm [hunk 7]
```diff
@@ -5,17 +5,14 @@
...
+	They are lighter and often more slender than men and bear fairy-like features upon their faces and ears.<br>\
+	The Elves worship the Divine Pantheon in the same manner as Humens, though they often practice a much older variation of the religion.<br>\
```
Comment (LonelyHowls) [inline code/modules/mob/living/carbon/human/species_types/roguetown/elf/elfs.dm:13]: "The Elves commonly worship the Divine Pantheon in much the same way that Humens do, though they tend towards older practices and beliefs about the Ten." or something like that

**Review by LonelyHowls — COMMENTED** [review]
These are just my suggestions! uwu

**Comment by Tree445** [comment]
Does this force skin color on wildkin?

**Comment by Kroto-Ker** [comment]
> Does this force skin color on wildkin?

It does not, the habitat selection doesn't force skin tone.

**Pull request merged**